### PR TITLE
Initial version of templates

### PR DESCRIPTION
**Pull request by Yolijn**
Interns at Frameless made some changes in their fork. You could consider squash merging as their commits are not yet perfect :)

- [x] Refactor template pages to use `tsx` instead of `mdx`
- [x] Add new components to template pages
- [x] Add new template pages

### CODE REVIEW

File: .storybook/stories/templates/02-Overview-page.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+type Story = StoryObj<typeof meta>;
+
+export const OverviewPage: Story = {
+  render: () => (
+    <Page>
+      <PageHeader>
+        <HeaderLogic {...overviewHeaderProps} />
+      </PageHeader>
+      <ResponsiveContent className="denhaag-page-content denhaag-responsive-content--sidebar">
+        <Sidenav>
```
Comment (Borai) [inline .storybook/stories/templates/02-Overview-page.stories.tsx:47]: Can we create a component for this?

File: .storybook/stories/templates/04-Current-affairs-page.stories.tsx
Comment (Borai) [inline .storybook/stories/templates/04-Current-affairs-page.stories.tsx:1]: Strange title maybe :P

**Review by Borai — COMMENTED** [review]
🚀

**Comment by YourivHDenHaag** [comment]
These commits are already merged: https://github.com/nl-design-system/denhaag/pull/1369

**Pull request closed without merging**